google-cloud-spanner/src/test/java/com/google/cloud/spanner/SpannerOptionsTest.java

```diff
@@ -954,6 +963,7 @@ public void testNumChannelsWithGrpcGcpExtensionEnabled() {
     SpannerOptions options2 =
         SpannerOptions.newBuilder()
             .setProjectId("test-project")
+            .setCredentials(NoCredentials.getInstance())
```

Comment:
This cleans up errors that are being logged during tests, clogging up the test log.